src/Compilers/CSharp/Test/Syntax/LexicalAndXml/PreprocessorTests.cs

```diff
@@ -1704,22 +1704,6 @@ class B { }
             VerifyMembers(node, new MemberInfo { Kind = SyntaxKind.ClassDeclaration, Text = "B" });
         }
 
-        [Fact]
-        [Trait("Feature", "Directives")]
-        public void TestNegIfEndifDirectivesWithBadCode()
```

Comment:
Why was this test deleted?

Reply:
> Why was this test deleted?

It looked like a dupe of a same named test in TopLevelStatementsParsingTests.cs, but I now realized that directives are not verified there. I will restore this aspect of the test.